cmd/timescale-prometheus/main.go

```diff
@@ -147,6 +152,7 @@ var (
 	writeThroughput     = util.NewThroughputCalc(tickInterval)
 	elector             *util.Elector
 	lastRequestUnixNano = time.Now().UnixNano()
+	safeVersion         = semver.Version{Major: 1, Minor: 7, Patch: 3}
```

Comment:
This should go into pkg/version/versions.go and be a semver range just like `ExtVersionRange `

Reply:
Done!